<!-- launch -->
<launch >
	<node name="simple_obstacle_avoidance" pkg="trajectory_tracker" type="simple_obstacle_avoidance" >
		<param name="/wheel_base" value="1.0" type="double" />	<!-- m -->
		<param name="/map_width" value="10.0" type="double" />	<!-- m -->
		<param name="/map_height" value="10.0" type="double" />	<!-- m -->
		<param name="/map_pixel" value="0.05" type="double" />	<!-- m -->
		<param name="/robot_width" value="1.0" type="double" />	<!-- m -->
		<param name="/predict_time" value="5.0" type="double" />	<!-- sec -->
		<param name="/step_time" value="0.1" type="double" />	<!-- sec -->
		<param name="/kv" value="0.8" type="double" />	<!-- must be included from 0 to 1 -->
		<param name="/w_range" value="0.3" type="double" />	<!-- rad/sec -->
		<param name="/source_twist_cmd_name" value="/twist_cmd" type="string" />
		<param name="/output_twist_cmd_name" value="/twist_cmd_with_avoidance" type="string" />
		<param name="/scan_name" value="/scan" type="string" />
		<param name="/use_emergency_stop" value="true" string="bool" />
		<param name="/stop_x_size" value="0.4" type="double" />	<!-- m -->
		<param name="/stop_y_size" value="0.6" type="double" />	<!-- m -->
		<param name="/stop_time" value="3.0" type="double" />	<!-- sec -->
		<param name="/back_time" value="5.0" type="double" />	<!-- sec -->
		<param name="/back_speed" value="0.2" type="double" />	<!-- m/sec -->
	</node >
</launch >
